website/content/ko/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "CNCF TAG 앱 제공"
+list_pages: true
+---
+
+<div class="row mt-5 mb-3">
+    <div class="col-lg-6">
+        <img src="/images/tag-app-delivery-horizontal-color.svg" alt="Tag 앱 제공 logo" style="max-width: 300px;">
+    </div>
+    <div class="col-lg-6">
+        <div class="lead">
+        구축, 패키징, 배포, 관리 및 운영을 포함하여 클라우드 네이티브 애플리케이션 제공과 관련된 프로젝트 및 계획입니다.
```

Comment:
Where is 'initiatives' ??? is not '계획' (plan)
And you may know KO's grammar is 

+ several '및' is not good writing.

Reply:
I've thought about it a lot... It looks like "초기 수립??" but I don't know how to express this with korean..  Google Translator interpreted it as "이니셔티브".

Reply:
Translating is for local people. Best sample is the movie of Deadpool. 
The point is fluent read and understand what it the meaning of this.

Reply:
Or I am highly recommend Kubernetes' KO translation team. 
https://kubernetes.slack.com/archives/CA1MMR86S
They are super good content in Korean. 
It could be great referencing.

Reply:
Thank you for suggesting. It is difficult to select appropriate words 😭 I will keep trying to find better word
Also how about changing initiatives with "단체"?? That makes sense than "계획". I will ask to them too!